samples/CMakeLists.txt

```diff
@@ -91,12 +91,12 @@ set(SAMPLES
     10_memcpy3D
     11_device
     hipStreamSemantics
-    hipMultiThreadAddCallback
+    #hipMultiThreadAddCallback
     hipInfo
     hipSymbol
     hip-cuda
     printf
-    texture
+    #texture
```

Comment:
This line was commented out in the `add graphs placeholders` patch. Was this intentional?

Reply:
Yes. It looked to me that these samples are the same as the unit tests since at least one of them had the same name.